### PR TITLE
Add eo-learn-geometry recipe

### PR DESCRIPTION
**Pull request by mwilson8**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mwilson8** [comment]
@conda-forge/staged-recipes @conda-forge/help-python I've never seen this error before; any reason why it's only on windows?

**Comment by chrisburr** [comment]
Maybe it's a random glitch...

@conda-forge-admin, please restart ci

**Comment by mwilson8** [comment]
@chrisburr any other thoughts?

**Comment by mwilson8** [comment]
@ocefpaf is this related to the `pillow` feedstock?

**Comment by mwilson8** [comment]
@dcunn @benhuff @oblute @rluria14 maintain this with me?

**Comment by mwilson8** [comment]
@conda-forge/staged-recipes @conda-forge/help-python please review

File: recipes/eo-learn-geometry/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win and vc<14 or py<35]
+  
+requirements:
+  host:
+    - pip
+    - python
+    - pillow
```
Comment (scopatz) [inline recipes/eo-learn-geometry/meta.yaml]: Is pillow really needed at install time?

**Comment by scopatz** [comment]
Still waiting on @benhuff

File: recipes/eo-learn-geometry/meta.yaml (hunk 1 above)
Comment (mwilson8) [reply recipes/eo-learn-geometry/meta.yaml]: probably not, left over from testing. removed in latest commit

**Comment by mwilson8** [comment]
@scopatz ready for merge

File: recipes/eo-learn-geometry/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - shapely
+    - gdal 
+    - fiona
+    - pillow
+test:
+  imports:
+    - eolearn
+    - eolearn.geometry
+  requires:
+    - pillow
```
Comment (scopatz) [inline recipes/eo-learn-geometry/meta.yaml]: This isn't needed because pillow is in the run deps

**Comment by mwilson8** [comment]
@scopatz please advise

**Comment by mwilson8** [comment]
@scopatz please review

**Comment by mwilson8** [comment]
@conda-forge/staged-recipes @conda-forge/core Is there a way to have the bots rebuild this feedstock so it isn't [empty](https://github.com/conda-forge/eo-learn-geometry-feedstock)??

**Comment by chrisburr** [comment]
There has been some issues with the feedstock conversion CI, see https://github.com/conda-forge/staged-recipes/issues/10438. The `eo-learn-geometry` folder is still in the master branch so it should finish being converted whenever the problem is fixed.

**Pull request merged**